cmd/user-agent/src/main.js

```diff
@@ -69,8 +69,8 @@ let defaultTrustBlocStartupOpts = {
     'log-level': 'debug',
     walletMediatorURL: 'https://localhost:10063',
     credentialMediatorURL: '',
-    agentUsername: 'user-agent',
-    sdsServerURL: 'https://localhost:8072/encrypted-data-vaults'
+    agentUsername: '',
+    sdsServerURL: ''
```

Comment:
Now that SDS is optional, there will be no errors if this is blank. Also note that we had decided that the default setting should be off while the edge-agent<->SDS flow is being developed.